### DISCUSSION TITLE
Hackaton: Road to Web3

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#20] [General]
> Road to Web3 is a purpose built event for you to create your first application on the Blockchain.
> 
> 
> Take your Web2 experience to new heights and build your first project interacting with decentralized games, DeFi, NFTs, or take something you already love in Web2 and decentralize it! We've gathered the best speakers, mentors, and thinkers in the space to help you learn and build with over 1000 new friends around the world.
> 
> 
> Any project deployed to Polygon will qualify for a guaranteed prize and if you build something great, there's thousands more up for grabs.

**Date: February 3 — 9, 2022**
https://web3.ethglobal.com/

<img width="965" alt="Road_to_Web3" src="https://user-images.githubusercontent.com/131172/151931032-9907c416-6f95-4e3f-a071-2de7c50df6c5.png">

**Comment by samnang** [comment]
Look at the schedule, People who live in the timezone +7 UTC need to stay all-night :)

![image](https://user-images.githubusercontent.com/131172/151931627-f507a2ac-f724-4fb4-b300-74df8d0f0803.png)

**Comment by babekasadli** [comment]
Thanks would like to join
